products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp

```diff
@@ -42,7 +42,9 @@ namespace app
     {
         getPopupFilter().addAppDependentFilter([&](const gui::PopupRequestParams &params) {
             const auto popupId = params.getPopupId();
-            if (popupId == gui::popup::ID::Alarm) {
+            if (popupId == gui::popup::ID::Alarm ||
```

Comment:
Won't this change also disable activation/deactivation popups on vertical layout?

Reply:
no, activation/deactivation views are handled independently in vertical layouts (they are not popups). I confirmed that on simulator.